Navigation.css

```diff
@@ -0,0 +1,82 @@
+body {
+    background-color: aliceblue;
```

Comment:
use variables for colors

Reply:
I agree. Once I merge it to my dev, I will be using the predefined variables for color.